source/plugin/blocks/accessing.rst

```diff
@@ -153,7 +154,7 @@ get the blast resistance of a block and checking if it is greater than or equal
 This will get the blast resistance of our block and compare it to a new :javadoc:`DoubleProperty`, as
 :javadoc:`BlastResistanceProperty` inherits from ``DoubleProperty``. The method will then return if the blast
 resistance of our block is greater than one, the value in placed ``matches()``. If we wanted to see if it was less than
-two, we would replace it with ``lessThan()``.
+two, we would replace it with ``Property.Operator.LESS``.
```

Comment:
What is "it"? It's not necessarily clear - specify that you can replace the property operator in the `DoubleProperty` ctor with this.